Formula/php.rb

```diff
@@ -113,50 +122,47 @@ def install
       --enable-sysvmsg
       --enable-sysvsem
       --enable-sysvshm
-      --enable-wddx
-      --enable-zip
       --with-apxs2=#{Formula["httpd"].opt_bin}/apxs
       --with-bz2#{headers_path}
-      --with-curl=#{Formula["curl-openssl"].opt_prefix}
+      --with-curl
+      --with-ffi
       --with-fpm-user=_www
       --with-fpm-group=_www
-      --with-freetype-dir=#{Formula["freetype"].opt_prefix}
-      --with-gd
+      --with-freetype
+      --with-gd=#{Formula["gd"].opt_prefix}
```

Comment:
```suggestion
```

`--with-gd` is gone.

Reply:
As of php/php-src@570d431, it's renamed to `--enable-gd`. Additionally, there are some more changes to the GD-related flags: php/php-src@48ca2c0. [Source](https://github.com/phpbrew/phpbrew/issues/1020)